cli/pkg/release/gb.go

```diff
@@ -23,25 +23,32 @@ func CreateGbPR(version, dir string) (gh.PullRequest, error) {
 
 	branch := "rnmobile/release_" + version
 
-	console.Info("Checking if branch %s exists", branch)
 	exists, _ := gh.SearchBranch("gutenberg", branch)
 
 	if (exists != gh.Branch{}) {
-		console.Info("Branch %s already exists", branch)
-		return pr, nil
+		console.Warn("Branch %s already exists", branch)
+
+		cont := console.Confirm("Do you wish to continue?")
```

Comment:
I like adding the confirm step here but we need more to get this working. 
On the other side of the `else` we are cloning the repo at `trunk` but if the branch exists we can clone off that. 

Also need to check if the PR already exists later on around line 133.

But all that can happen in a follow up PR
If you want we can return a "not implemented" error if someone confirms that working with the existing branch is ok. I'm not worried about that if we start on working out the existing branch work next